pkg/kubelet/kubelet.go

```diff
@@ -1400,7 +1401,7 @@ func (kl *Kubelet) setupDataDirs() error {
 	if err := os.MkdirAll(kl.getRootDir(), 0750); err != nil {
 		return fmt.Errorf("error creating root directory: %v", err)
 	}
-	if err := os.MkdirAll(kl.getPodLogsDir(), 0750); err != nil {
+	if err := utilfs.MkdirAll(kl.getPodLogsDir(), 0750); err != nil {
```

Comment:
there are ~124 non-test calls to MkdirAll across ~88 files ... are all of those wrong?

Reply:
what are the implications of only changing a few of these calls to the fixed ones?

Reply:
for the immediate need we have (for the patch release deadline tomorrow!) i think we are ok with the current state, but yes, i'd support auditing once more / again.

Reply:
I do plan on looking into other areas (specifically volume mount propagations) as a follow-up to this work.
That work is complicated by the fact that all containers running as `ContainerUser` have the same effective security context for Windows containers tho :(

Windows doesn't currently have a way to ACL anything on the host in such a way that we would be able to give access to a **specific** container running as `ContainerUser`